web/api/report_server.thrift

```diff
@@ -396,6 +396,17 @@ struct CheckerCount {
 }
 typedef list<CheckerCount> CheckerCounts
 
+struct CheckerInfo {
+  1: string       checkerName,  // Analyzer name of the checker.
+  2: string       analyzerName, // Analyzer name of the checker.
+  3: Severity     severity,     // Severity level of the checker.
+  4: list<i64> enabled,      // Runs' names in which the checker enabled.
+  5: list<i64> disabled,     // Runs' names in which the checker disabled.
```

Comment:
Formatting is wrong, also, "Run names" is enough, no need to add a genitive.

Reply:
But wait, you're saying you're returning run names here, but the returned type is list of ints. So what is returned here? Maybe it got turned into Run IDs?

Reply:
You're right. I didn't update the comments properly. Enabled and disabled list are run id lists not names.